[es] Actualiza los metadatos y mejora la redacción en la plantilla de… (#35499)
mdn/translated-content: files/es/mdn/writing_guidelines/page_structures/page_types/css_property_page_template/index.md

title: Plantilla de página de propiedad CSS
slug: MDN/Writing_guidelines/Page_structures/Page_types/CSS_property_page_template
l10n:
  sourceCommit: d2fb8cdc9422dd2b68ff23f616d70811729f1fbd
---

> {{SeeCompatTable}}{{deprecated_header}}

Reviewer: Dos issues en esta línea/zona:

**(a) Macro casing y falta `{{Non-standard_Header}}`.** El fuente en inglés usa:

```
{{SeeCompatTable}}{{Deprecated_Header}}{{Non-standard_Header}}
```

Mientras que aquí está como `{{SeeCompatTable}}{{deprecated_header}}` (minúsculas y sin Non-standard).

```suggestion
{{SeeCompatTable}}{{Deprecated_Header}}{{Non-standard_Header}}
```

**(b) Falta todo el bloque `[!NOTE]` explicativo del fuente en inglés (líneas 8-77 de `files/en-us/.../index.md`)**. Ese bloque documenta `title`, `slug`, `page-type`, `status`, `browser-compat`, `sidebar`, las macros de encabezado y las macros `{{CSSSyntax}}` / `{{CSSInfo}}`. Aunque indica "_Remove this note block before publishing._", esa instrucción aplica a quien instancia la plantilla, no al archivo de plantilla en sí. El archivo en `mdn/content` la conserva, por lo que la versión en español también debe hacerlo. Por favor, agrega el bloque traducido entre el front-matter y la línea de macros.

Comienza el contenido de la página con un párrafo introductorio que nombre la propiedad y explique qué hace. Idealmente, debería ser una o dos frases cortas.

> ## Pruébalo (Try it)

Reviewer: El inglés es solo `## Try it`. La forma `## Pruébalo (Try it)` añade el nombre original entre paréntesis dentro del encabezado, lo cual no es habitual en otras plantillas/páginas traducidas.

```suggestion
## Pruébalo
```

Y actualizar también la referencia en la línea 14: `Incluye el título de la sección "Pruébalo"`.

title: Plantilla de página de propiedad CSS
slug: MDN/Writing_guidelines/Page_structures/Page_types/CSS_property_page_template
l10n:
  sourceCommit: d2fb8cdc9422dd2b68ff23f616d70811729f1fbd
---

> [!NOTE]
> _Elimina este bloque de notas antes de publicar._
>
> ---
>
> **Front matter de la página:**
>
> El front matter en la parte superior de la página se utiliza para definir los "metadatos de la página".
> Los valores deben actualizarse adecuadamente para la propiedad en particular.
>
> ```md
> ---
> title: name-of-the-property
> slug: Web/CSS/Reference/Properties/name-of-the-property
> page-type: css-property O css-shorthand-property
> status:
>   - deprecated
>   - experimental
>   - non-standard
> browser-compat: css.properties.name-of-the-property
> sidebar: cssref
> ---
> ```
>
> - **title**
>   - : El valor de `title` se muestra en la parte superior de la página. El formato del título es _nombre-de-la-propiedad_.
>     Por ejemplo, la propiedad {{cssxref("background-color")}} tiene el título _background-color_.
> - **slug**

> >   - : El valor de `slug` es el final de la ruta de la URL después de `[https://developer.mozilla.org/en-US/docs/](https://developer.mozilla.org/en-US/docs/)`. Esto tendrá el formato `Web/CSS/Reference/Properties/nombre-de-la-propiedad`.

Reviewer: **Nuevo — URL dentro de backticks con formato Markdown (línea 36).** Dentro de comillas invertidas (`` ` ``), la sintaxis Markdown no se procesa, así que `[url](url)` se muestra como texto literal en lugar de la URL sola. Debe quedar solo la URL:

```suggestion
>   - : El valor de `slug` es el final de la ruta de la URL después de `https://developer.mozilla.org/en-US/docs/`. Esto tendrá el formato `Web/CSS/Reference/Properties/nombre-de-la-propiedad`.
```

> >   - : El valor de `slug` es el final de la ruta de la URL después de `[https://developer.mozilla.org/en-US/docs/](https://developer.mozilla.org/en-US/docs/)`. Esto tendrá el formato `Web/CSS/Reference/Properties/nombre-de-la-propiedad`.

Reviewer: **Línea 36 — URL dentro de backticks con formato Markdown.** Dentro de comillas invertidas la sintaxis Markdown no se procesa, así que `[url](url)` se muestra como texto literal. El inglés usa la URL directamente:

```suggestion
>   - : El valor de `slug` es el final de la ruta de la URL después de `https://developer.mozilla.org/en-US/docs/`. Esto tendrá el formato `Web/CSS/Reference/Properties/nombre-de-la-propiedad`.
```

>
> ---
>
> **Otras macros en la página**
>
> - Sección de sintaxis formal: El contenido para la sección _Sintaxis formal_ se genera utilizando la macro `\{{CSSSyntax}}`. Esta macro obtiene datos de las especificaciones utilizando el [paquete npm @webref/css](https://www.npmjs.com/package/@webref/css).
> - Sección de definición formal: El contenido para la sección _Definición formal_ se genera utilizando la macro `\{{CSSInfo}}`. Para que esta sección tenga datos, debes asegurarte de que se haya completado una entrada adecuada para la propiedad correspondiente en el archivo de datos [properties.json](https://github.com/mdn/data/blob/main/css/properties.json) en el repositorio `mdn/data`. Consulta la página de [Propiedades](https://github.com/mdn/data/blob/main/css/properties.md) para obtener más información.
> - Secciones de Especificaciones y Compatibilidad con navegadores: La herramienta de compilación utiliza automáticamente el par clave-valor `browser-compat` del front matter de la página para insertar datos en las secciones _Especificaciones_ y _Compatibilidad con navegadores_ (reemplazando las macros `\{{Specifications}}` y `\{{Compat}}` en esas secciones, respectivamente).
>

> >   Ten en cuenta que es posible que primero debas crear o actualizar una entrada para la propiedad y su especificación en nuestro <a href="[https://github.com/mdn/browser-compat-data](https://github.com/mdn/browser-compat-data)">repositorio de datos de compatibilidad de navegadores</a>.

Reviewer: **Nuevo — href roto (línea 74).** El atributo `href` contiene sintaxis Markdown `[url](url)` dentro de HTML, lo que genera un enlace roto. El inglés usa la URL directamente:

```suggestion
>   Note that you may first need to create/update an entry for the property and its specification in our <a href="https://github.com/mdn/browser-compat-data">repositorio de datos de compatibilidad de navegadores</a>.
```

> >   Ten en cuenta que es posible que primero debas crear o actualizar una entrada para la propiedad y su especificación en nuestro <a href="[https://github.com/mdn/browser-compat-data](https://github.com/mdn/browser-compat-data)">repositorio de datos de compatibilidad de navegadores</a>.

Reviewer: **Línea 74 — `href` roto.** El atributo `href` contiene la sintaxis Markdown `[url](url)` dentro de un atributo HTML, lo que genera un enlace roto en el navegador. La fuente en inglés usa la URL directamente:

```suggestion
>   Ten en cuenta que es posible que primero debas crear o actualizar una entrada para la propiedad y su especificación en nuestro <a href="https://github.com/mdn/browser-compat-data">repositorio de datos de compatibilidad de navegadores</a>.
```

>   Consulta nuestra [guía de tablas de compatibilidad](/es/docs/MDN/Writing_guidelines/Page_structures/Compatibility_tables) para obtener información sobre cómo añadir o editar entradas.
>
> _Recuerda eliminar este bloque de notas antes de publicar._

> {{SeeCompatTable}}{{deprecated_header}}{{Non-standard_Header}}

Reviewer: Línea 79: `{{deprecated_header}}` debe ir con mayúsculas iniciales como en la fuente en inglés: `{{Deprecated_Header}}`. Yari es sensible a mayúsculas en los nombres de macro.

```suggestion
{{SeeCompatTable}}{{Deprecated_Header}}{{Non-standard_Header}}
```

> {{SeeCompatTable}}{{deprecated_header}}{{Non-standard_Header}}

Reviewer: **Pendiente de revisiones anteriores (línea 79).** `{{deprecated_header}}` debe ir con mayúsculas iniciales, igual que en la fuente en inglés. Yari es sensible a mayúsculas en los nombres de macro.

```suggestion
{{SeeCompatTable}}{{Deprecated_Header}}{{Non-standard_Header}}
```

> {{SeeCompatTable}}{{deprecated_header}}{{Non-standard_Header}}

Reviewer: **Línea 79 — `{{deprecated_header}}` debe ir con mayúsculas iniciales.** Yari es sensible a mayúsculas en los nombres de macro. La fuente en inglés usa `{{Deprecated_Header}}`.

```suggestion
{{SeeCompatTable}}{{Deprecated_Header}}{{Non-standard_Header}}
```

> Enlaces a más ejemplos en otras páginas
> ```
>
> **Escenario 2:** Si _solo_ tienes ejemplos en otra página y ninguno en esta:
>
> No añadas encabezados H3; simplemente añade los enlaces directamente bajo el encabezado H2 "Ejemplos". Por ejemplo:
>
> ```md
> ## Ejemplos
>
> Para ver ejemplos de esta API, consulta [la página sobre fetch()](https://example.org/).
> ```

## Especificaciones

`\{{Specifications}}`

_Para usar esta macro, elimina las comillas invertidas y la barra invertida en el archivo markdown._

## Compatibilidad con navegadores

`\{{Compat}}`

_Para usar esta macro, elimina las comillas invertidas y la barra invertida en el archivo markdown._

> ## Ver también

Reviewer: Terminología: `See also` → `Véase también` según el glosario MDN-es.

```suggestion
## Véase también
```

Y actualizar también el texto del enlace en la línea 114 a `[sección Véase también]`.

> Enlaces a más ejemplos en otras páginas
> ```
>
> **Escenario 2:** Si _solo_ tienes ejemplos en otra página y ninguno en esta:
>
> No añadas encabezados H3; simplemente añade los enlaces directamente bajo el encabezado H2 "Ejemplos". Por ejemplo:
>
> ```md
> ## Ejemplos
>
> Para ver ejemplos de esta API, consulta [la página sobre fetch()](https://example.org/).
> ```

## Especificaciones

`\{{Specifications}}`

_Para usar esta macro, elimina las comillas invertidas y la barra invertida en el archivo markdown._

## Compatibilidad con navegadores

`\{{Compat}}`

_Para usar esta macro, elimina las comillas invertidas y la barra invertida en el archivo markdown._

## Véase también

> Incluye enlaces a páginas de referencia y guías relacionadas con la propiedad actual. Para más pautas, consulta la [sección Ver también](/es/docs/MDN/Writing_guidelines/Writing_style_guide#see_also_section) en la _Guía de estilo de escritura_.

Reviewer: Línea 185: por consistencia con el encabezado `## Véase también` de la línea 183, el texto del enlace debería ser `[sección Véase también]` en lugar de `[sección Ver también]`.

```suggestion
Incluye enlaces a páginas de referencia y guías relacionadas con la propiedad actual. Para más pautas, consulta la [sección Véase también](/es/docs/MDN/Writing_guidelines/Writing_style_guide#see_also_section) en la _Guía de estilo de escritura_.
```

> Incluye enlaces a páginas de referencia y guías relacionadas con la propiedad actual. Para más pautas, consulta la [sección Ver también](/es/docs/MDN/Writing_guidelines/Writing_style_guide#see_also_section) en la _Guía de estilo de escritura_.

Reviewer: **Pendiente de revisión anterior (línea 185).** Por consistencia con el encabezado `## Véase también` de la línea 183, el texto del enlace debe ser `[sección Véase también]` en lugar de `[sección Ver también]`.

```suggestion
Incluye enlaces a páginas de referencia y guías relacionadas con la propiedad actual. Para más pautas, consulta la [sección Véase también](/es/docs/MDN/Writing_guidelines/Writing_style_guide#see_also_section) en la _Guía de estilo de escritura_.
```

> Incluye enlaces a páginas de referencia y guías relacionadas con la propiedad actual. Para más pautas, consulta la [sección Ver también](/es/docs/MDN/Writing_guidelines/Writing_style_guide#see_also_section) en la _Guía de estilo de escritura_.

Reviewer: **Línea 185 — texto del enlace inconsistente.** El encabezado de la línea 183 dice `## Véase también`, pero el texto del enlace sigue diciendo `[sección Ver también]`. Deben coincidir:

```suggestion
Incluye enlaces a páginas de referencia y guías relacionadas con la propiedad actual. Para más pautas, consulta la [sección Véase también](/es/docs/MDN/Writing_guidelines/Writing_style_guide#see_also_section) en la _Guía de estilo de escritura_.
```
